Makefile

```diff
@@ -305,31 +294,38 @@ create-cluster-management: $(CLUSTERCTL) ## Create a development Kubernetes clus
 	kubectl \
 		--kubeconfig=$$(kind get kubeconfig-path --name="clusterapi") \
 		create -f examples/_out/controlplane.yaml
-	# Get KubeConfig using clusterctl.
-	$(CLUSTERCTL) \
-		alpha phases get-kubeconfig -v=3 \
-		--kubeconfig=$$(kind get kubeconfig-path --name="clusterapi") \
-		--namespace=default \
-		--cluster-name=$(CLUSTER_NAME)
-	# Apply addons on the target cluster, waiting for the control-plane to become available.
-	$(CLUSTERCTL) \
-		alpha phases apply-addons -v=10 \
-		--kubeconfig=./kubeconfig \
-		-a examples/addons.yaml
 	# Create a worker node with MachineDeployment.
 	kubectl \
 		--kubeconfig=$$(kind get kubeconfig-path --name="clusterapi") \
 		create -f examples/_out/machinedeployment.yaml
 
+	# wait for all machines to become ready
+	./examples/wait-for-ready.sh
+
+	# Fetch the Kubeconfig for the target cluster
+	source ./examples/_out/.env; \
+	kubectl \
+		--kubeconfig=$$(kind get kubeconfig-path --name="clusterapi") \
+		--namespace=default get secret/$$CLUSTER_NAME-kubeconfig -o json \
+		| jq -r .data.value \
+		| base64 --decode > ./examples/_out/clusterapi.kubeconfig
+
+	# Deploy a CNI soution, Calico
+	kubectl --kubeconfig=./examples/_out/clusterapi.kubeconfig \
+      apply -f https://docs.projectcalico.org/v3.8/manifests/calico.yaml
+
+	@echo 'run "kubectl --kubeconfig=$$(kind get kubeconfig-path --name="clusterapi") ..." to work with the kind cluster'
```

Comment:
should this be 
```suggestion
	@echo 'run "kubectl --kubeconfig=$(kind get kubeconfig-path --name="clusterapi")" to work with the kind cluster'
```

Reply:
This is how I see the above on the cmd line output:
```
run "kubectl --kubeconfig=$(kind get kubeconfig-path --name="clusterapi") ..." to work with the kind cluster
```

The extra `$` is eaten up by `make`.

Reply:
yeah you're right, what about the `...` at the end?